src/pages/Profile/Profile.tsx

```diff
@@ -4,9 +4,23 @@ import { Link } from "react-router-dom";
 import { UserProfile } from "../../mockData/mockTribe";
 import Header from "../../components/Header/Header";
 import Navigation from "../../components/Navigation/Navigation";
+import { signOut } from "firebase/auth";
+import { auth } from "../../firebase";
+import { useNavigate } from "react-router-dom";
 
 const Profile = ({ user }: { user: UserProfile }) => {
   const { totalScore, img, name, bio, email } = user;
+  const navigate = useNavigate();
+
+  const handleSignOut = () => {
+    signOut(auth)
```

Comment:
Can you write this using async await syntax, please?

You can use a try / catch for the error